papertty.py

```diff
@@ -131,6 +131,16 @@ def img_diff(img1, img2):
     def ttydev(vcsa):
         """Return associated tty for vcsa device, ie. /dev/vcsa1 -> /dev/tty1"""
         return vcsa.replace("vcsa", "tty")
+    
+    def vcsudev(self, vcsa):
```

Comment:
Doesn't matter that much to me, but this way we can keep it static:

```suggestion
    @staticmethod
    def vcsudev(vcsa, force_vcs=False):
```

Reply:
I'm actually not very happy about all those static methods, especially `font_height` and `set_tty_size`, whose first parameter when they are run is a field of PaperTTY. It would be a bit more object-oriented to make them instance methods and use `self`. But that is a topic for later.

I also don't really see a go reason to force vcs mode unless vcsu isn't available or the font doesn't support it, i.e. is not a TrueType font, but that's something the PaperTTY class already knows. That suggests an instance method again.

Reply:
I think you're right - instance method it is then.